src/test/java/wooteco/subway/dao/line/LineDaoTest.java

```diff
@@ -0,0 +1,113 @@
+package wooteco.subway.dao.line;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.transaction.annotation.Transactional;
+import wooteco.subway.domain.line.Line;
+
+@SpringBootTest(webEnvironment = WebEnvironment.MOCK)
+@Transactional
+class LineDaoTest {
+
+    @Autowired
+    @Qualifier("jdbc")
+    private LineDao lineDao;
+
+    @Test
+    void dependency() {
+        assertThat(lineDao).isNotNull();
+    }
+
+    @Test
+    void save() {
+        // given
+        Line line = new Line("3호선", "bg-blue-500");
+
+        // when
+        Line persistedLine = lineDao.save(line);
+
+        // then
+        assertAll(
+            () -> assertThat(line.getName()).isEqualTo(persistedLine.getName()),
+            () -> assertThat(line.getColor()).isEqualTo(persistedLine.getColor())
+        );
+    }
+
+    @Test
+    void findAll() {
+        // given
+        Line line1 = new Line("3호선", "bg-blue-600");
+        Line line2 = new Line("4호선", "bg-yellow-600");
+
+        // when
+        lineDao.save(line1);
+        lineDao.save(line2);
+        List<Line> lines = lineDao.findAll();
+
+        // then
+        assertAll(
+            () -> assertThat(lines.get(0).getName()).isEqualTo(line1.getName()),
+            () -> assertThat(lines.get(0).getColor()).isEqualTo(line1.getColor()),
+            () -> assertThat(lines.get(1).getName()).isEqualTo(line2.getName()),
+            () -> assertThat(lines.get(1).getColor()).isEqualTo(line2.getColor())
+        );
```

Comment:
```suggestion
        assertThat(lines)
                .extracting("name", "color")
                .containsExactlyInAnyOrder(
                        tuple(line1.getName(), line1.getColor()),
                        tuple(line2.getName(), line2.getColor())
                );
```

어떤 리스트의 값을 검증하고자 할 때는 이런 방법도 있으니 참고해보세요 :)